ui/app/mainui/AppMain.qml

```diff
@@ -724,6 +726,14 @@ Item {
                     closeBtnVisible: false
                 }
 
+                ModuleWarning {
+                    id: mailserverConnectionBanner
+                    type: ModuleWarning.Danger
```

Comment:
I thing Warning is fine here. The app will still function without a mailserver

Reply:
Okey. And what color we are using for Warnings? `pinColor1` or i should add one to palette?

Reply:
Oh I assumed we already supported Warning. It should be orange, but someone from the UI team can answer cc @caybro

Reply:
<img width="1399" alt="Screenshot 2023-01-18 at 20 36 14" src="https://user-images.githubusercontent.com/2522130/213240336-0b75970b-92d0-4d3d-87c3-b8aa463a85cf.png">
<img width="1396" alt="Screenshot 2023-01-18 at 20 36 26" src="https://user-images.githubusercontent.com/2522130/213240366-b1b08abd-241d-46d8-8502-21cacbdb16d1.png">

This is orange color from StatusColors. Imho it's too peachy

Reply:
Yeah that orange is not great. There is `warning` which uses another `orange` in Light and DarkTheme. I'm not sure if those theme files are still in use, but that orange is better IMO

Reply:
That orange is [#FFA500](https://www.color-hex.com/color/ffa500). It's better than peach, but not as good as [International Orange](
https://en.wikipedia.org/wiki/International_orange) :)
Maybe i can replace old orange name with `peach` and add orange as warning color?

Reply:
@MishkaRogachev you know we can use directly the CSS named colors? ;) https://doc.qt.io/qt-6/qml-color.html#svg-color-reference

Reply:
But I'd leave the decision up to @benjthayer or @John-44

Reply:
@caybro I've poked @benjthayer @filipedesign @pedro-et to reply here with the correct orange colour value, agreed this orange is too peachy.  You will get a reply about the correct colour value to use here very soon

Reply:
Hey @caybro apologies for the delay - I've worked up an orange semantic colour for desktop and rolled that out in the Discord Import tool for consistency.

The value is #F6793C

![RAG-notification](https://user-images.githubusercontent.com/110033914/213435957-1af278aa-2323-432d-90fc-cf8eac50c765.png)

 Thanks

Reply:
<img width="1326" alt="Screenshot 2023-01-19 at 17 51 55" src="https://user-images.githubusercontent.com/2522130/213460079-a2c45afc-92d4-49b8-985b-5ad405806671.png">
<img width="1328" alt="Screenshot 2023-01-19 at 17 51 39" src="https://user-images.githubusercontent.com/2522130/213460096-f3f3a914-e910-4b92-a9f1-7ed644495f70.png">
 One color both for light and dark themes